sonar-jproperties-plugin/src/main/java/org/sonar/plugins/jproperties/issuesaver/DuplicatedKeysAcrossFilesIssueSaver.java

```diff
@@ -30,19 +30,9 @@
 import org.sonar.plugins.jproperties.api.visitors.issue.IssueLocation;
 import org.sonar.plugins.jproperties.api.visitors.issue.PreciseIssue;
 
-class CrossFileChecks {
+public class DuplicatedKeysAcrossFilesIssueSaver {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1118)